bindings/java/native/src/bee_types/index.rs

```diff
@@ -0,0 +1,36 @@
+use anyhow::anyhow;
+use crate::Result;
+
+use iota_wallet::{
+    message::{
+        IndexationPayload as IndexationPayloadRust,
+    }
+};
+
+pub struct IndexationPayload {
+    payload: IndexationPayloadRust,
+}
+
+impl IndexationPayload {
+    pub fn get_internal(self) -> IndexationPayloadRust {
```

Comment:
there's a bunch of functions called `get_internal`; in Rust the idiomatic name would be `to_inner`.